html/user-activation/resources/utils.js

```diff
@@ -1,37 +1,56 @@
+/**
+ * Calls the function (f) after N number of frames have passed.
+ *
+ * @param {Function} f
+ * @param {number} num_frames
+ */
 function delayByFrames(f, num_frames) {
   function recurse(depth) {
-    if (depth == 0)
-      f();
-    else
-      requestAnimationFrame(() => recurse(depth-1));
+    if (depth == 0) f();
```

Comment:
Not a big fan of this style.

Reply:
yeah, I'm not a huge fan either... was just keeping the original. I'll see about adding { } in a followup.